v22.2/alter-index.md

```diff
@@ -73,3 +85,192 @@ Subcommand | Description
+    -------------+-----------------------------------------------+------------+--------------+---------------+-----------+---------+----------+----------
+      rides      | rides_auto_index_fk_city_ref_users            |     t      |            1 | city          | ASC       |    f    |    f     |    t
+      rides      | rides_auto_index_fk_city_ref_users            |     t      |            2 | rider_id      | ASC       |    f    |    f     |    t
+      rides      | rides_auto_index_fk_city_ref_users            |     t      |            3 | id            | ASC       |    f    |    t     |    t
+      rides      | rides_auto_index_fk_vehicle_city_ref_vehicles |     t      |            1 | vehicle_city  | ASC       |    f    |    f     |    t
+      rides      | rides_auto_index_fk_vehicle_city_ref_vehicles |     t      |            2 | vehicle_id    | ASC       |    f    |    f     |    t
+      rides      | rides_auto_index_fk_vehicle_city_ref_vehicles |     t      |            3 | city          | ASC       |    f    |    t     |    t
+      rides      | rides_auto_index_fk_vehicle_city_ref_vehicles |     t      |            4 | id            | ASC       |    f    |    t     |    t
+      rides      | rides_pkey                                    |     f      |            1 | city          | ASC       |    f    |    f     |    t
+      rides      | rides_pkey                                    |     f      |            2 | id            | ASC       |    f    |    f     |    t
+      ...
+      rides      | rides_pkey                                    |     f      |           10 | revenue       | N/A       |    t    |    f     |    t
+    (17 rows)
+    ~~~
+
+1. Explain the query that filters on revenue. Since there is no index on the `revenue` column, the query performs a full scan.
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    EXPLAIN SELECT * FROM rides WHERE revenue > 90 ORDER BY revenue ASC;
+    ~~~
+
+    ~~~
+                                                                           info
+    ---------------------------------------------------------------------------------------------------------------------------------------------------
+      distribution: full
+      vectorized: true
+
+      • sort
+      │ estimated row count: 12,417
+      │ order: +revenue
+      │
+      └── • filter
+          │ estimated row count: 12,417
+          │ filter: revenue > 90
+          │
+          └── • scan
+                estimated row count: 125,000 (100% of the table; stats collected 4 minutes ago)
+                table: rides@rides_pkey
+                spans: FULL SCAN
+
+      index recommendations: 1
+      1. type: index creation
+         SQL command: CREATE INDEX ON rides (revenue) STORING (vehicle_city, rider_id, vehicle_id, start_address, end_address, start_time, end_time);
+    (19 rows)
+    ~~~
+
+1. Create the recommended index.
```

Comment:
`1.` -> `3.`? and I guess all the following bulletin number should be in a sequence :)